### DISCUSSION TITLE
Feature Request: Ability to start with last defined position and size

### DISCUSSION THREAD

**Discussion by DesctructBear** [tom-englert/RegionToShare#7] [Ideas]
Great tool, really handy for large monitors!
Would love to see an option to start the app with the last position and size of the window (region). Another possibility would be app settings to manually set the data (e.g. left=900 top=1 width=1920 height=1080), this could also allow multiple presets (if you have a 38" at home and a 34" in the office). Keep up the great work!

Cheers
NeDrY

**Comment by tom-englert** [comment]
I'm using it with [Fancy Zones](https://docs.microsoft.com/en-us/windows/powertoys/fancyzones), so I had no real need for this. 
However I will consider this for a next release.

**Comment by tom-englert** [comment]
See also #11

**Comment by Ib2cool** [comment]
Thanks for a really great tool
Lock (no resize) to fx. 1920x1080 at startup, would make this tool even better
Simple shortcut parameter would do it

**Comment by tom-englert** [comment]
See #11

**Comment by DesctructBear** [comment]
Great, thanks! Works, but if I end the program in presentation mode (right clicking and choose close window) then the program is shrunk on next startup (width -8px, height -4px).
